context.go

```diff
@@ -188,61 +180,84 @@ func (c *Ctx) LoadProject() (*Project, error) {
 	return p, nil
 }
 
-// resolveProjectRoot evaluates the root directory and does the following:
-//
-// If the passed path is a symlink outside GOPATH to a directory within a
-// GOPATH, the resolved full real path is returned.
-//
-// If the passed path is a symlink within a GOPATH, we return an error.
-//
-// If the passed path isn't a symlink at all, we just pass through.
-func (c *Ctx) resolveProjectRoot(path string) (string, error) {
-	// Determine if this path is a Symlink
-	l, err := os.Lstat(path)
-	if err != nil {
-		return "", errors.Wrap(err, "resolveProjectRoot")
+/*
```

Comment:
i'm just now realizing that some of the existing comments did this. that's my bad - i shouldn't have merged those in the first place. still, let's not repeat the pattern.